yargs.js

```diff
@@ -301,42 +301,58 @@ function Yargs (processArgs, cwd, parentRequire) {
     return self
   }
 
-  self.demand = self.required = self.require = function (keys, max, msg) {
+  self.demandOption = self.demand = self.required = self.require = function (keys, max, msg) {
```

Comment:
If I get this right, `demandOption` only accepts specific option names instead of just an integer for a number of minimum options, like `demandCommand`, right? (Even though the title of `demandOption` still says `count` as a parameter)

In that case, `demandOption` _does not replace `demand`_, which is why I think `demand` should be removed even as an alias, because that could cause confusion.